doc/sphinx/language/coq-library.rst

```diff
@@ -6,22 +6,34 @@ The Coq libraries
 .. index::
    single: Theories
 
-The :gdef:`prelude` contains definitions and theorems for
+The :gdef:`core library` contains definitions and theorems for
 the most commonly used elementary logical notions and
-data types. Coq loads many of these files automatically when it starts.
-The content of the prelude can be browsed at https://coq.inria.fr/prelude/.
+data types, as well as :ref:`primitive objects <primitive-objects>`,
+support for tactics such as :tacn:`setoid_rewrite`
+or commands such as :ref:`Program <programs>`, the proof language
+:ref:`ssreflect <thessreflectprooflanguage>`,
+the tactic language :ref:`Ltac2 <ltac2>`
+and some compatibilty helpers for previous versions of Rocq.
+The content of the corelib can be browsed at https://rocq-prover.org/corelib/.
+
+Coq loads many of these files automatically when it starts.
+This set of preloaded files is called the :gdef:`prelude`.
+Its content can be browsed at
+https://rocq-prover.org/corelib/Corelib.Init.Prelude.html.
 
 Other libraries like the :gdef:`standard library` are general-purpose
 libraries with definitions and theorems for sets, lists, sorting,
 arithmetic, etc. To use these files, users must load them explicitly
-with the ``Require`` command (see :ref:`compiled-files`)
-The content of the standard library can be browsed at https://coq.inria.fr/stdlib/.
+with the ``From Stdlib Require`` command (see :ref:`compiled-files`).
+The content of the standard library can be browsed at https://rocq-prover.org/stdlib/.
 
-There are also many libraries provided by Coq users' community.
-These libraries and developments are available
-for download at https://coq.inria.fr/ (see :ref:`userscontributions`).
+There are many other libraries provided by Rocq users' community.
+Many such libraries and developments are packaged in Nix or Opam.
+The Rocq Opam repository can be browsed at
+https://rocq-prover.org/packages/
+(see :ref:`userscontributions`).
```

Comment:
```suggestion
There are many other libraries provided by the Rocq users' community.
Many such libraries and developments are packaged in Nix or Opam.
The Rocq Opam repository can be browsed at
https://rocq-prover.org/packages/
(see :ref:`userscontributions`).
```

Also nix is mentioned, but not linked. I don't know the best place to link.

Reply:
Not sure either what would be the best thing to link here.